doc/planar.xml

```diff
@@ -0,0 +1,314 @@
+    <P/>
+    
+    <C>IsPlanarDigraph</C> returns <K>true</K> if the digraph <A>digraph</A> is
+    planar and <K>false</K> if it is not. The directions and multiplicities of
+    any edges in <A>digraph</A> are ignored by <C>IsPlanarDigraph</C>.
+    <P/> 
+
+    See also <Ref Prop="IsOuterPlanarDigraph"/>.
+    <P/>
+
+    This method uses the reference implementation in
+    &edge-addition-planarity-suite; by John Boyer of the algorithms described
+    in <Cite Key="Boyer2006aa"/>.
+
+    <Example><![CDATA[
+gap> IsPlanarDigraph(CompleteDigraph(4));
+true
+gap> IsPlanarDigraph(CompleteDigraph(5));
+false
+gap> IsPlanarDigraph(CompleteBipartiteDigraph(2, 3));
+true
+gap> IsPlanarDigraph(CompleteBipartiteDigraph(3, 3));
+false
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="IsOuterPlanarDigraph">
+<ManSection>
+  <Prop Name="IsOuterPlanarDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    An <E>outer planar</E> digraph is a digraph that can be embedded in the
+    plane in such a way that its edges do not intersect, and all vertices
+    belong to the unbounded face of the embedding.  A digraph is planar if and
+    only if it does not have a subdigraph that is homeomorphic to either the
+    complete graph on <C>4</C> vertices or the complete bipartite graph with
+    vertex sets of sizes <C>2</C> and <C>3</C>.
+    <P/>
+    
+    <C>IsOuterPlanarDigraph</C> returns <K>true</K> if the digraph
+    <A>digraph</A> is outer planar and <K>false</K> if it is not. The
+    directions and multiplicities of any edges in <A>digraph</A> are ignored by
+    <C>IsPlanarDigraph</C>.  <P/>
+
+    See also <Ref Prop="IsPlanarDigraph"/>.
+
+    This method uses the reference implementation in
+    &edge-addition-planarity-suite; by John Boyer of the algorithms described
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> IsOuterPlanarDigraph(CompleteDigraph(4));
+false
+gap> IsOuterPlanarDigraph(CompleteDigraph(5));
+false
+gap> IsOuterPlanarDigraph(CompleteBipartiteDigraph(2, 3));
+false
+gap> IsOuterPlanarDigraph(CompleteBipartiteDigraph(3, 3));
+false
+gap> IsOuterPlanarDigraph(CycleDigraph(10));
+true
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="KuratowskiPlanarSubdigraph">
+<ManSection>
+  <Attr Name="KuratowskiPlanarSubdigraph" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    <C>KuratowskiPlanarSubdigraph</C> returns the list of out-neighbours
+    of a (not necessarily induced) subdigraph of the digraph <A>digraph</A>
+    that witness the fact that <A>digraph</A> is not planar, or
+    <K>fail</K> if <A>digraph</A> is planar.  In other words,
+    <C>KuratowskiPlanarSubdigraph</C> returns the out-neighbours of a
+    subdigraph of <A>digraph</A> that is homeomorphic to the complete graph
+    with four vertices, or to the complete bipartite graph with vertex sets of
```

Comment:
Don't you mean K_5 and K_3,3 and not K_4 and K_2,3?

Reply:
Also note that here you talk about `the complete graph with four vertices` whereas above you talk about the `complete graph on <C>4</C> vertices` - it's not vital but I'd recommend to use the same wording.